### ISSUE TITLE
Missing comma in footer for float values

### ISSUE THREAD

**Issue by Issif** [olekukonko/tablewriter#93]
Hi,

When I print a float value in a footer, comma just disappears. You have exact same trouble in Output 2 of your README : https://github.com/olekukonko/tablewriter/blob/master/README.md#output-2

```
    DATE   |       DESCRIPTION        |  CV2  | AMOUNT
+----------+--------------------------+-------+---------+
  1/1/2014 | Domain name              |  2233 | $10.98
  1/1/2014 | January Hosting          |  2233 | $54.95
  1/4/2014 | February Hosting         |  2233 | $51.00
  1/4/2014 | February Extra Bandwidth |  2233 | $30.00
+----------+--------------------------+-------+---------+
                                        TOTAL | $146 93
                                      +-------+---------+]
```

Regards,

**Comment by mattn** [comment]
I'm not sure why amtFmt is applied to footer, but you can fix behavior with call `SetAutoFormatHeaders`.
```
table.SetAutoFormatHeaders(false)
```

**Comment by mattn** [comment]
Closed by #95
